src/views/product-tutorials-view/server.ts

```diff
@@ -41,6 +46,58 @@ export interface ProductPageData {
 	inlineTutorials: InlineTutorials
 }
 
+/**
+ * Note: this is a stub to get the /hcp/tutorials rendering.
+ * TODO: figure out what to do with the /hcp/tutorials view (design dependent).
+ * Taking this temporary approach for now while awaiting final designs.
+ */
+export async function getCloudTutorialsViewProps() {
```

Comment:
As mentioned above, this is a stub and not necessarily meant as the final props for the HCP page. But hopefully it's a good start!